metron-interface/metron-alerts/scripts/run-all-test.sh

```diff
@@ -14,10 +15,21 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
-port: 4200
+set -e
 
-rest:
-  host: localhost
-  port: 4200
+current_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+cd "$current_path/.."
 
-dirPath: dist
\ No newline at end of file
+source ~/.nvm/nvm.sh; nvm install; node --version
+
+npm ci
+
+npm run test:ci
+npm run cypress:ci
+
+# licence check needs an initial build step
+npm run build
+echo "┌─────────────────────────────────────┐"
+echo "| Checking Apache license headers...  |"
+echo "└─────────────────────────────────────┘"
+./scripts/prepend_license_header.sh
```

Comment:
For some reason, git recognized this change as a rename. It is actually a removal of a yml config file (belongs to our previous test server setup, no longer needed) and the addition of a shell scirpt.